src/ontology/edits/oeo-shared.omn

```diff
@@ -2494,6 +2494,33 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1390",
         OEO_00000014
     
     
+Class: OEO_00010361
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A renewable power unit is a power generating unit that has renewable energy as input.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue:https://github.com/OpenEnergyPlatform/ontology/issues/1423
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1437",
+        rdfs:label "renewable power unit"@en
+    
+    EquivalentTo: 
+        OEO_00000334
+         and (OEO_00010234 some OEO_00020085)
```

Comment:
```suggestion
         and (OEO_00010234 only OEO_00020085)
```
Some or only?
Or rather focus on the output of energy here?

Reply:
Hm, I am not sure whether we should change this axiom. Some renewable power generating units also use small amount of fossil energy. E.g. in a [Zündstrahlmotor](https://de.wikipedia.org/wiki/Z%C3%BCndstrahlmotor) tiny amounts of diesel fuel are used to facilitate the combustion of biogas.

But maybe we can neglect that?

Reply:
This works also the other way round: is a power plant with a minor share of renewable energy input renewable?

Reply:
I like the idea you had today in the OEO dev meeting to introduce here a function. Something like a `renewable energy harvest function`: _A renewable energy harvest function is an energy transformation function of an artificial object that has been engineered to transform renewable energy._ Then we could leave the energy input axiom as it is.

This only downside is that energy transformations function will be only added when we implement #1438. One way around this is to finalise this PR now and add the axiom `'renewable power unit' 'has function' some 'renewable harvest function'` as part of #1438.

Reply:
PR #1438 is merged now. I suggest that we change the definition of `renewable power generating unit` from _A renewable power unit is a power generating unit that has renewable energy as input._ To: _A renewable power unit is a power generating unit that has the function to transform renewable energy._
This would then be inline with what we intend to do in PR #1360.

The axioms should be:
```
'renewable power unit' EquivalentTo: 'power generating unit' and ('has function' some 'renewable energy function')
'renewable power unit' SubclassOf: 'has energy input' some 'renewable energy'
```

Reply:
Go ahead!